src/IB/IBFEMethod.cpp

```diff
@@ -820,7 +860,9 @@ IBFEMethod::forwardEulerStep(const double current_time, const double new_time)
     const double dt = new_time - current_time;
     for (unsigned int part = 0; part < d_num_parts; ++part)
     {
-        int ierr = VecWAXPY(d_X_new_vecs[part]->vec(), dt, d_U_current_vecs[part]->vec(), d_X_current_vecs[part]->vec());
+        if (!d_part_is_active[part]) continue;
```

Comment:
Let's not include this guard --- I think that I would like for all computations to proceed "as usual" for inactivated parts, except that at the end of the day, forces are not spread and velocity is not interpolated.